src/Uno.UWP/System/Power/PowerManager.wasm.cs

```diff
@@ -0,0 +1,132 @@
+﻿#nullable enable
+
+
+using System;
+using Uno;
+using Uno.Foundation;
+
+namespace Windows.System.Power;
+
+partial class PowerManager
+{
+	private const string JsType = "Windows.System.Power.PowerManager";
+
+	private static bool _isInitialized;
+	private static bool _isChargingChangedSubscribed;
+
+	static partial void StartPowerSupplyStatus() => StartOnChargingChange();
+
+	static partial void EndPowerSupplyStatus() => EndOnChargingChange();
+
+	static partial void StartBatteryStatus() => StartOnChargingChange();
+
+	static partial void EndBatteryStatus() => EndOnChargingChange();
+
+	private static void StartOnChargingChange()
+	{
+		if (_isChargingChangedSubscribed)
+		{
+			return;
+		}
+
+		WebAssemblyRuntime.InvokeJS($"{JsType}.startChargingChange()");
```

Comment:
This shouldn't be the way to go.

Reply:
Indeed, it is an old PR and I failed to realize that it is now done with introp 😅